### ISSUE TITLE
Task settings revert to default

### ISSUE THREAD

**Issue by manybodycpa** [opencv/cvat#1209]
After, for example, enabling autosave and setting the framerate for playback, the settings go back to the default values after the task is closed and reopened. Is this the expected behaviour?

**Comment by bsekachev** [comment]
@manybodycpa 
It is expected, but we can consider it as a feature request

**Comment by manybodycpa** [comment]
Thanks for the reply. I would like to kindly request an option to remember the task settings if others find that feature useful.

**Comment by manybodycpa** [comment]
Alternatively, is there a way to just change the default settings for all tasks for a given user?

**Comment by bsekachev** [comment]
Related with #747

**Comment by bsekachev** [comment]
@manybodycpa 

No way except of changing source code.

**Comment by cpudney** [comment]
Yes, this would be a useful enhancement. We have to enable autosave for each annotation job whereas we'd like this setting persisted indefinately.

There are other preferences we'd like persisted too, e.g. how annotations are coloured (I prefer by class) and opacity. Should I put this in a separate issue?

![image](https://user-images.githubusercontent.com/1069553/89882027-d62f0700-dbf8-11ea-8fa3-d6ecc1f1a129.png)

Thanks for a great tool.

**Comment by TheRealRolandDeschain** [comment]
Hi everybody, we stumbled across this issue since we have the same problem, that we want to set and retrieve the user settings (especially autosave) on the REST server. 

In our case registering users, setting up annotation jobs and so on is done automatically by our own server which talks to CVAT's REST Server, the actual users should only work on the annotation itself. After registering a new user we would also like to turn on the automatic saving feature for each user, which is not possible, if these settings are only stored on the client. 

I'm not quite sure how #3017 solves this particular issue, but we might be missing something here that would solve our problem.
